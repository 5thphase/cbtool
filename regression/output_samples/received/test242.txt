VM object DD958086-32D4-5B48-9558-8A8EC2E9A164 (named "vm_3") had its run state successfully changed to "resume" on this experiment.
The following state values are available for the objects on this experiment (Cloud TESTCLOUD) :
------------------ PER-OBJECT STATE (CLOUD TESTCLOUD)  -----------------
|Object Type    |Object Name    |Object UUID                             |State                     
|VMC            |simzone_b      |49282BA1-6D05-5C43-9AF0-FFE568AFD5F9    |attached                  
|VMC            |simzone_a      |F04BDEFB-2E30-560C-802C-A1FABC5D8D0C    |attached                  
|VMC            |simzone_d      |9C8B8765-2143-577F-9E2E-DEDF96009D33    |attached                  
|VMC            |simzone_c      |68F26DD5-96FF-5310-9ED9-3568A17CBF4A    |attached                  
|VMC            |simzone_b      |49282BA1-6D05-5C43-9AF0-FFE568AFD5F9    |attached                  
|VMC            |simzone_a      |F04BDEFB-2E30-560C-802C-A1FABC5D8D0C    |attached                  
|VMC            |simzone_d      |9C8B8765-2143-577F-9E2E-DEDF96009D33    |attached                  
|VMC            |simzone_c      |68F26DD5-96FF-5310-9ED9-3568A17CBF4A    |attached                  
|HOST           |host_simhostb4 |000DDB49-8618-55C0-9D8D-FDF5956DE270    |attached                  
|HOST           |host_simhostc4 |7BDABEFB-8C06-587A-AADF-9267A04EA683    |attached                  
|HOST           |host_simhostb0 |D30E646F-CD41-56BC-92B9-945251F61D20    |attached                  
|HOST           |host_simhostc2 |2FEFED3D-66EA-5AA2-ADB5-87DB7F60F87A    |attached                  
|HOST           |host_simhostd1 |9E9AD6B1-0FAC-540A-B9F4-3111F83AD0FC    |attached                  
|HOST           |host_simhostb1 |5DB9BC6C-1A82-5493-90A5-F9B5E7DDC279    |attached                  
|HOST           |host_simhostc0 |21CFA660-9E7F-5DAC-83E7-93EE3497463F    |attached                  
|HOST           |host_simhostc1 |32FE9370-67B2-57D7-88F0-A51B02CE22D7    |attached                  
|HOST           |host_simhosta1 |0A64BD54-9DEC-5F8A-BEB0-16D8FF787363    |attached                  
|HOST           |host_simhosta4 |A544E0A7-5991-5B7C-A698-A4CDE09ECF49    |attached                  
|HOST           |host_simhosta2 |068C8C7E-A166-5FCF-B9CD-615F964FC172    |attached                  
|HOST           |host_simhosta0 |257E4891-BCEB-554D-A9AA-714703241467    |attached                  
|HOST           |host_simhostc3 |2F3DE71E-672D-5BF3-A759-B248D26AD0F7    |attached                  
|HOST           |host_simhosta3 |9E9B48DC-2C08-5970-AD0E-8300B19153B9    |attached                  
|HOST           |host_simhostb3 |EC11C436-1219-5628-8BA8-A4310470DBDA    |attached                  
|HOST           |host_simhostd4 |71C8BA33-79AE-5D40-8141-CF3155CC5B22    |attached                  
|HOST           |host_simhostd2 |DBFA9943-E424-5278-81AA-EF48AF3B49C9    |attached                  
|HOST           |host_simhostd3 |D441925C-2230-5FD9-A6D5-055362AE5A2E    |attached                  
|HOST           |host_simhostb2 |1D443863-3DE2-5F73-A9F9-F2A156F9DB2D    |attached                  
|HOST           |host_simhostd0 |337028CC-52ED-5FF3-8AEC-22C3E740DF5B    |attached                  
|VMC            |simzone_b      |49282BA1-6D05-5C43-9AF0-FFE568AFD5F9    |attached                  
|VMC            |simzone_a      |F04BDEFB-2E30-560C-802C-A1FABC5D8D0C    |attached                  
|VMC            |simzone_d      |9C8B8765-2143-577F-9E2E-DEDF96009D33    |attached                  
|VMC            |simzone_c      |68F26DD5-96FF-5310-9ED9-3568A17CBF4A    |attached                  
|HOST           |host_simhostb4 |000DDB49-8618-55C0-9D8D-FDF5956DE270    |attached                  
|HOST           |host_simhostc4 |7BDABEFB-8C06-587A-AADF-9267A04EA683    |attached                  
|HOST           |host_simhostb0 |D30E646F-CD41-56BC-92B9-945251F61D20    |attached                  
|HOST           |host_simhostc2 |2FEFED3D-66EA-5AA2-ADB5-87DB7F60F87A    |attached                  
|HOST           |host_simhostd1 |9E9AD6B1-0FAC-540A-B9F4-3111F83AD0FC    |attached                  
|HOST           |host_simhostb1 |5DB9BC6C-1A82-5493-90A5-F9B5E7DDC279    |attached                  
|HOST           |host_simhostc0 |21CFA660-9E7F-5DAC-83E7-93EE3497463F    |attached                  
|HOST           |host_simhostc1 |32FE9370-67B2-57D7-88F0-A51B02CE22D7    |attached                  
|HOST           |host_simhosta1 |0A64BD54-9DEC-5F8A-BEB0-16D8FF787363    |attached                  
|HOST           |host_simhosta4 |A544E0A7-5991-5B7C-A698-A4CDE09ECF49    |attached                  
|HOST           |host_simhosta2 |068C8C7E-A166-5FCF-B9CD-615F964FC172    |attached                  
|HOST           |host_simhosta0 |257E4891-BCEB-554D-A9AA-714703241467    |attached                  
|HOST           |host_simhostc3 |2F3DE71E-672D-5BF3-A759-B248D26AD0F7    |attached                  
|HOST           |host_simhosta3 |9E9B48DC-2C08-5970-AD0E-8300B19153B9    |attached                  
|HOST           |host_simhostb3 |EC11C436-1219-5628-8BA8-A4310470DBDA    |attached                  
|HOST           |host_simhostd4 |71C8BA33-79AE-5D40-8141-CF3155CC5B22    |attached                  
|HOST           |host_simhostd2 |DBFA9943-E424-5278-81AA-EF48AF3B49C9    |attached                  
|HOST           |host_simhostd3 |D441925C-2230-5FD9-A6D5-055362AE5A2E    |attached                  
|HOST           |host_simhostb2 |1D443863-3DE2-5F73-A9F9-F2A156F9DB2D    |attached                  
|HOST           |host_simhostd0 |337028CC-52ED-5FF3-8AEC-22C3E740DF5B    |attached                  
|VM             |vm_3           |DD958086-32D4-5B48-9558-8A8EC2E9A164    |attached                  
|VM             |vm_14          |A6BE1AB6-F90E-5F1D-8BAF-C029580346F0    |attached                  
|VM             |vm_4           |7EA82D09-5944-5F5F-8854-5CE385071C35    |attached                  
|VM             |vm_9           |21F523CA-5683-5874-8DFF-FE0B3E68B77E    |attached                  
|VM             |vm_16          |38E550F9-B6EE-5E04-B3B0-505C8B4046E1    |attached                  
|VM             |vm_10          |2AB6AB34-A741-50D2-9F09-BD31BA45C80C    |attached                  
|VM             |vm_13          |9E1BE97A-DD71-5BDB-9930-9903AA014075    |attached                  
|VM             |vm_6           |4467B178-CAEC-521A-96DD-477E02D7A7DB    |attached                  
|VM             |vm_1           |6ACE62BD-D7F9-5736-8A15-14EFCDB4D3C8    |attached                  
|VM             |vm_19          |A477B2AF-E8E2-5036-8515-F48CF4B0851C    |attached                  
|VM             |vm_22          |F4E55870-AD0C-50E9-88D3-4EA1D6144A89    |attached                  
|VM             |vm_5           |9A9038FF-525E-5953-A91F-BA8B51E0629D    |attached                  
|VM             |vm_18          |15052811-FFAE-5704-AB00-FFDDA1AB76AC    |attached                  
|VM             |vm_8           |B57980AF-27E4-5B16-95D2-A3C9C8709AC6    |attached                  
|VM             |vm_17          |648637A9-2DED-5BE7-8642-C6CE843565B3    |attached                  
|VM             |vm_21          |809642D0-08AF-5793-92BC-C4FD5023A60A    |attached                  
|VM             |vm_7           |E83EE478-7F8E-5E8D-B913-82EAF311CE79    |attached                  
|VM             |vm_12          |B1D912F0-2781-50DE-A02A-AB857CB9A235    |attached                  
|VM             |vm_15          |520DCF94-E762-5CFD-8EC5-33ADC8097D49    |attached                  
|VM             |vm_20          |0587EEDA-5A3B-548C-BE72-4095113C207B    |attached                  
|VM             |vm_11          |4392830F-431A-5757-BEF2-789A49CD997D    |attached                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |1.76M                         
|Redis Version                                        |2.6.16                        
|Uptime (in seconds)                                  |507659                        
|Total Connections Received                           |10480                         
|Total Commands Processed                             |591023                        
|Number of Keys                                       |427                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.9                         
|Storage Size                                         |1282048                       
|Data Size                                            |161728                        
|Index Size                                           |294336                        
|Average Object Size                                  |402.308457711                 
|Collections                                          |14                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |21                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |21                            
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |111                           

 ---------------- VMC-WIDE COUNTERS ----------------
|49282BA1-6D05-5C43-9AF0-FFE568AFD5F9 (simzone_b) VM RESERVATIONS|7                             
|F04BDEFB-2E30-560C-802C-A1FABC5D8D0C (simzone_a) VM RESERVATIONS|6                             
|9C8B8765-2143-577F-9E2E-DEDF96009D33 (simzone_d) VM RESERVATIONS|1                             
|68F26DD5-96FF-5310-9ED9-3568A17CBF4A (simzone_c) VM RESERVATIONS|7                             

