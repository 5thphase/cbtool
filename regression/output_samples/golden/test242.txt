VM object 549ADC5F-81E3-537E-AC36-359C8C8582CE (named "vm_3") had its run state successfully changed to "resume" on this experiment.
The following state values are available for the objects on this experiment (Cloud TESTCLOUD) :
------------------ PER-OBJECT STATE (CLOUD TESTCLOUD)  -----------------
|Object Type    |Object Name    |Object UUID                             |State                     
|VMC            |simzone_a      |2A3C7A6F-F507-5C26-894B-CA777DA4A067    |attached                  
|VMC            |simzone_c      |841649F1-B9B9-5F66-BF08-8188BB2F57BF    |attached                  
|VMC            |simzone_d      |290AA171-A379-5C01-9211-06AFF9C329FA    |attached                  
|VMC            |simzone_b      |DDE459C8-87C4-5C23-B346-48A22B829E97    |attached                  
|VMC            |simzone_a      |2A3C7A6F-F507-5C26-894B-CA777DA4A067    |attached                  
|VMC            |simzone_c      |841649F1-B9B9-5F66-BF08-8188BB2F57BF    |attached                  
|VMC            |simzone_d      |290AA171-A379-5C01-9211-06AFF9C329FA    |attached                  
|VMC            |simzone_b      |DDE459C8-87C4-5C23-B346-48A22B829E97    |attached                  
|HOST           |host_simhostc2 |6DC4F441-50F3-506B-834A-706B4362CAAE    |attached                  
|HOST           |host_simhostb1 |63A1C42F-7561-5658-A92E-DF6164774886    |attached                  
|HOST           |host_simhostb3 |498CE010-12A6-5DD7-9857-B4EB16A71F7F    |attached                  
|HOST           |host_simhostc4 |BC02B3CF-498E-59BB-9099-35225D5FB5FA    |attached                  
|HOST           |host_simhostb4 |2FA25AB3-AB5E-5341-9865-846D7458CA54    |attached                  
|HOST           |host_simhostd0 |8567C2FC-5C30-58FB-A72B-181718F80AFE    |attached                  
|HOST           |host_simhostb2 |10E0E690-5CBE-5148-BB5E-16CAD04AC3F3    |attached                  
|HOST           |host_simhostd2 |1B324D97-237B-50DE-AD76-E48669D56BB3    |attached                  
|HOST           |host_simhostd3 |0E22BD3E-B5E6-5CC3-B6ED-0DB3F65E354C    |attached                  
|HOST           |host_simhostd1 |8FE194F2-2426-54A5-A56C-B29D7453F30E    |attached                  
|HOST           |host_simhostb0 |ADFD7B06-47C4-5570-92A6-A12C9C6A2C2F    |attached                  
|HOST           |host_simhosta2 |6A1C0996-942E-5131-8CAB-72ED75ADD99B    |attached                  
|HOST           |host_simhostd4 |3558BE2D-4F01-5A92-8B16-550C63F2F495    |attached                  
|HOST           |host_simhosta1 |FE109B99-DDB9-5BCB-B3C4-86A722A826A3    |attached                  
|HOST           |host_simhosta0 |E9C11D2D-03C8-5B86-B18E-E87265D6F0A8    |attached                  
|HOST           |host_simhosta4 |7C4CAA84-AF75-53C4-A665-49BC8E484998    |attached                  
|HOST           |host_simhostc0 |CEE5533E-6B32-5AD8-973F-B8B1CCEC8966    |attached                  
|HOST           |host_simhosta3 |D6088D27-EC47-5944-9EE9-FBCCBF83D26F    |attached                  
|HOST           |host_simhostc1 |5845837D-97E2-5A8D-A1CA-9CACDABF8C0D    |attached                  
|HOST           |host_simhostc3 |1AEEB9A4-806C-5B75-BE61-976B7075CE6F    |attached                  
|VMC            |simzone_a      |2A3C7A6F-F507-5C26-894B-CA777DA4A067    |attached                  
|VMC            |simzone_c      |841649F1-B9B9-5F66-BF08-8188BB2F57BF    |attached                  
|VMC            |simzone_d      |290AA171-A379-5C01-9211-06AFF9C329FA    |attached                  
|VMC            |simzone_b      |DDE459C8-87C4-5C23-B346-48A22B829E97    |attached                  
|HOST           |host_simhostc2 |6DC4F441-50F3-506B-834A-706B4362CAAE    |attached                  
|HOST           |host_simhostb1 |63A1C42F-7561-5658-A92E-DF6164774886    |attached                  
|HOST           |host_simhostb3 |498CE010-12A6-5DD7-9857-B4EB16A71F7F    |attached                  
|HOST           |host_simhostc4 |BC02B3CF-498E-59BB-9099-35225D5FB5FA    |attached                  
|HOST           |host_simhostb4 |2FA25AB3-AB5E-5341-9865-846D7458CA54    |attached                  
|HOST           |host_simhostd0 |8567C2FC-5C30-58FB-A72B-181718F80AFE    |attached                  
|HOST           |host_simhostb2 |10E0E690-5CBE-5148-BB5E-16CAD04AC3F3    |attached                  
|HOST           |host_simhostd2 |1B324D97-237B-50DE-AD76-E48669D56BB3    |attached                  
|HOST           |host_simhostd3 |0E22BD3E-B5E6-5CC3-B6ED-0DB3F65E354C    |attached                  
|HOST           |host_simhostd1 |8FE194F2-2426-54A5-A56C-B29D7453F30E    |attached                  
|HOST           |host_simhostb0 |ADFD7B06-47C4-5570-92A6-A12C9C6A2C2F    |attached                  
|HOST           |host_simhosta2 |6A1C0996-942E-5131-8CAB-72ED75ADD99B    |attached                  
|HOST           |host_simhostd4 |3558BE2D-4F01-5A92-8B16-550C63F2F495    |attached                  
|HOST           |host_simhosta1 |FE109B99-DDB9-5BCB-B3C4-86A722A826A3    |attached                  
|HOST           |host_simhosta0 |E9C11D2D-03C8-5B86-B18E-E87265D6F0A8    |attached                  
|HOST           |host_simhosta4 |7C4CAA84-AF75-53C4-A665-49BC8E484998    |attached                  
|HOST           |host_simhostc0 |CEE5533E-6B32-5AD8-973F-B8B1CCEC8966    |attached                  
|HOST           |host_simhosta3 |D6088D27-EC47-5944-9EE9-FBCCBF83D26F    |attached                  
|HOST           |host_simhostc1 |5845837D-97E2-5A8D-A1CA-9CACDABF8C0D    |attached                  
|HOST           |host_simhostc3 |1AEEB9A4-806C-5B75-BE61-976B7075CE6F    |attached                  
|VM             |vm_9           |3E36F938-309E-5FB8-B897-C0A515A80F49    |attached                  
|VM             |vm_10          |AED8F3E8-44E8-572A-B5E0-C4FF30B3D1B1    |attached                  
|VM             |vm_15          |CB48D46F-13E7-5DBC-8C05-997C903D4D03    |attached                  
|VM             |vm_6           |4261B402-7054-59D7-B5E3-BA084BF4235A    |attached                  
|VM             |vm_19          |C4AA0E1C-9AEC-551E-9FF8-DEDBBEF3B6FF    |attached                  
|VM             |vm_1           |142CA977-1346-51A4-8723-15A0C834B452    |attached                  
|VM             |vm_20          |5E4EABD9-D0F0-54A5-88BC-AE8A432EC6CC    |attached                  
|VM             |vm_18          |332867FF-015B-5E8E-B777-3B562B90DF32    |attached                  
|VM             |vm_22          |BECFE961-199E-50F1-8DA4-E9551B70F640    |attached                  
|VM             |vm_3           |549ADC5F-81E3-537E-AC36-359C8C8582CE    |attached                  
|VM             |vm_8           |4DC44809-7FAC-5785-9A48-41C29B9EAAC9    |attached                  
|VM             |vm_21          |B443178B-F66D-51A3-A7C9-E5239FE9D54F    |attached                  
|VM             |vm_4           |06168A73-1A99-5C3A-A075-8B1543B64634    |attached                  
|VM             |vm_5           |BA7874EC-CC1A-5A3F-B994-BB10A4B0982E    |attached                  
|VM             |vm_17          |A6450C57-8808-5368-9273-810B41E79659    |attached                  
|VM             |vm_14          |221831B6-345A-565F-A351-6D0AB465AC39    |attached                  
|VM             |vm_13          |3221797E-AE68-50F0-B59F-B23851C7E12A    |attached                  
|VM             |vm_11          |1A9CE402-BA50-5717-B0ED-9843C8642E90    |attached                  
|VM             |vm_12          |709B110A-55AC-59B7-B35F-A8C89D6BD03F    |attached                  
|VM             |vm_16          |72248C73-FCB6-5B45-B171-92B35880938D    |attached                  
|VM             |vm_7           |0F0EBE49-2C77-560B-8E35-461587455150    |attached                  

The following statistics are available on this experiment (Cloud TESTCLOUD) :
------------------------- OBJECT STORE -----------------------
|Metric                                               |Value                         
|Used Memory                                          |2.40M                         
|Redis Version                                        |2.5.12                        
|Uptime (in seconds)                                  |2416600                       
|Total Connections Received                           |4440                          
|Total Commands Processed                             |3654113                       
|Number of Keys                                       |427                           
------------------------- METRIC STORE -----------------------
|Metric                                               |Value                         
|MongoDB Version                                      |2.4.5                         
|Storage Size                                         |5447958528.0                  
|Data Size                                            |4960663224.0                  
|Index Size                                           |133612192                     
|Average Object Size                                  |8110.56230728                 
|Collections                                          |40                            
--------------------- EXPERIMENT OBJECTS ---------------------
|Object                                               |Count                         
|VMCs                                                 |4                             
|HOSTs                                                |20                            
|VMs                                                  |21                            
|AIs                                                  |0                             
|AIDRSs                                               |0                             
|VMCRSs                                               |0                             
|FIRSs                                                |0                             
------------------ EXPERIMENT-WIDE COUNTERS ------------------
|Counter                                              |Value                         
|VMC RESERVATIONS                                     |4                             
|VMCs ARRIVED                                         |16                            
|VMCs ARRIVING                                        |0                             
|VMCs DEPARTED                                        |12                            
|VMCs DEPARTING                                       |0                             
|VMCs FAILED                                          |0                             
|HOST RESERVATIONS                                    |0                             
|HOSTs ARRIVED                                        |0                             
|HOSTs ARRIVING                                       |0                             
|HOSTs DEPARTED                                       |0                             
|HOSTs DEPARTING                                      |0                             
|HOSTs FAILED                                         |0                             
|VM RESERVATIONS                                      |21                            
|VMs ARRIVED                                          |22                            
|VMs ARRIVING                                         |0                             
|VMs CAPTURING                                        |0                             
|VMs DEPARTED                                         |1                             
|VMs DEPARTING                                        |0                             
|VMs FAILED                                           |0                             
|AI RESERVATIONS                                      |0                             
|AIs ARRIVED                                          |0                             
|AIs ARRIVING                                         |0                             
|AIs CAPTURING                                        |0                             
|AIs DEPARTED                                         |0                             
|AIs DEPARTING                                        |0                             
|AIs FAILED                                           |0                             
|AIDRS RESERVATIONS                                   |0                             
|AIDRSs ARRIVED                                       |0                             
|AIDRSs ARRIVING                                      |0                             
|AIDRSs DEPARTED                                      |0                             
|AIDRSs DEPARTING                                     |0                             
|AIDRSs FAILED                                        |0                             
|VMCRS RESERVATIONS                                   |0                             
|VMCRSs ARRIVED                                       |0                             
|VMCRSs ARRIVING                                      |0                             
|VMCRSs DEPARTED                                      |0                             
|VMCRSs DEPARTING                                     |0                             
|VMCRSs FAILED                                        |0                             
|FIRS RESERVATIONS                                    |0                             
|FIRSs ARRIVED                                        |0                             
|FIRSs ARRIVING                                       |0                             
|FIRSs DEPARTED                                       |0                             
|FIRSs DEPARTING                                      |0                             
|FIRSs FAILED                                         |0                             
|EXPERIMENT COUNTER                                   |111                           

 ---------------- VMC-WIDE COUNTERS ----------------
|2A3C7A6F-F507-5C26-894B-CA777DA4A067 (simzone_a) VM RESERVATIONS|7                             
|841649F1-B9B9-5F66-BF08-8188BB2F57BF (simzone_c) VM RESERVATIONS|7                             
|290AA171-A379-5C01-9211-06AFF9C329FA (simzone_d) VM RESERVATIONS|1                             
|DDE459C8-87C4-5C23-B346-48A22B829E97 (simzone_b) VM RESERVATIONS|6                             

